PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,9 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            //Issue 2 Resolution:
+            //Cast the int parametes to doubles to avoid integer division. Tests are completing successfully.
+            return ((1.0 * numberCorrectAnswers) / (1.0 * numberOfQuestions) * 100) + "%";
```

Comment:
You could have just cast one of the numbers to a float as well, which then would have made all following operations float instead of int. The only other change I would have made would have been to add a round in case the answer was not a whole number
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```